bpf/lib/nodeport.h

```diff
@@ -571,7 +571,7 @@ int tail_nodeport_ipv6_dsr(struct __ctx_buff *ctx)
 }
 #endif /* ENABLE_DSR */
 
-__section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
+declare_tailcall_if(__not(is_defined(IS_BPF_LXC)), CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
```

Comment:
I'm not sure if this would be better but please consider using positive condition and if that's not possible then maybe defining positive condition separately, e.g.

```
declare_tailcall_if(is_defined(X_ENABLED), CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
declare_tailcall_if(is_defined(IS_BPF_ABC), CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
```
or
```
#define LOAD_NODEPORT_NAT_INGRESS (conidtions)
declare_tailcall_if(is_defined(LOAD_NODEPORT_NAT_INGRESS), CILIUM_CALL_IPV6_NODEPORT_NAT_INGRESS)
```